### DISCUSSION TITLE
What do the matches list mean?

### DISCUSSION THREAD

**Discussion by t-j-thomas** [shazamio/ShazamIO#55] [Q&A]
First of all, thank you for this great project!

Sorry if this is a dumb question, but what can I infer from the matches list returned for a song recognition? For some requests, there are many matches, and for others there is just one.

Does a match list of size 1 mean the confidence is higher than if there were more matches? Or is it the other way round? Or should I not even interpret this in any way?

Also, does the `location` field with the accuracy have any meaning? From what I have seen, the accuracy value is always 0.01

Thanks!

<img width="649" alt="Screenshot 2023-04-22 202847" src="https://user-images.githubusercontent.com/127390988/233816249-7d758cee-a2b1-4e18-b86f-1647525bde99.png">

**Comment by dotX12** [comment]
@t-j-thomas hi!

The melody is divided into several sections. In simple terms, this track could be defined in this way:
there is a match from 1 to 3 seconds
from 8 to 10 there is a match
from 15 to 18 seconds there is a match.
matches is responsible for this and in fact you don't need it at all :)

Sometimes there can be several values, sometimes one, this is just the specifics of shazam, sometimes one match is enough for him to accurately determine, sometimes not, but he gives information on which sections he was able to accurately determine what kind of track it was.

**Comment by dotX12** [comment]
Regarding `location accuracy`, I will not give exact information for what he is responsible for)
